### DISCUSSION TITLE
Q: Pre moderation comment

### DISCUSSION THREAD

**Discussion by githubhabIT** [walinejs/waline#2531] [Q&A] [answered]
### Describe the bug | 问题描述

How do I make comments published not immediately? 

### Website URL | 问题网站

https://xboxrussia.tilda.ws/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

How do I make comments published not immediately?

### Website URL | Problem website

https://xboxrussia.tilda.ws/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
https://waline.js.org/en/reference/server/env.html#safety `COMMENT_AUDIT` set to `true` and redeploy your app.
